mmedit/models/editors/stylegan3/stylegan3_modules.py

```diff
@@ -169,19 +173,21 @@ def __init__(self,
                 activation='lrelu',
                 lr_multiplier=lr_multiplier)
             setattr(self, f'fc{idx}', layer)
-        self.register_buffer('w_avg', torch.zeros([style_channels]))
+        if num_ws is not None and w_avg_beta is not None:
```

Comment:
may checkout whether have this condition

Reply:
Conditional discriminator set `num_ws` as `None` in its `MappingNetwork`. When `num_ws` is `None`, the output style code will not be repeated (shape like `(bz, style_channels)`). Otherwise, the output will shape like `(bz, num_ws, style_channels)`. 
This logical follows StyleGAN3's implementation.